src/pages/editor.js

```diff
@@ -53,8 +55,7 @@ const initBridge = function (key, sbResolveRelations, setStory) {
       sbClient
         .get(`cdn/stories/${getParam('path')}`, {
           version: 'draft',
-          resolve_relations: sbResolveRelations || [],
-          token: key,
```

Comment:
This doesn't work then eh? Perhaps a suggestion should go back to them to update their docs.

Reply:
I'll let our bestie, Hannes, know! 😎